common/extensions/api/brave_shields.json

```diff
@@ -64,6 +64,39 @@
           }
         ]
       },
+      {
```

Comment:
you don't need any of this. If we're just setting the actual pref value, that is handled for you. The extra methods for brave shields defaults are there because we don't even use the pref values

Reply:
How do you mean @bridiver? What method exists within the shields extension for changing this new preference (`kAdvancedViewControlType`)?